sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -59,7 +59,7 @@ private[spark] object HiveUtils extends Logging {
   val isHive23: Boolean = hiveVersion.startsWith("2.3")
 
   /** The version of hive used internally by Spark SQL. */
-  val builtinHiveVersion: String = if (isHive23) hiveVersion else "1.2.1"
+  val builtinHiveVersion: String = if (isHive23) "2.3.5" else "1.2.1"
```

Comment:
https://github.com/apache/spark/pull/25443#issuecomment-521078239 - yeah, otherwise people might keep asking why it's 2.3.5.